README.md

```diff
@@ -1 +1,82 @@
-# angular-track-by-readme
\ No newline at end of file
+# Track by
+
+## Overview
+
+Track by allows Angular to speed up our `ng-repeat`s massively - let's take a look how.
+
+## Objectives
+
+- Describe "track by" clause
+- Write an ng-repeat that utilises "track by"
+
+## What is track by?
+
+Track by allows us to tell Angular what to identify unique items in our arrays, so Angular can tell when to remove it from the DOM correctly.
+
+If we were to have an `ng-repeat` repeating items of news, Angular will display all of the news items on the page.
+
+Later on, if we were to refresh the data (refresh the news), Angular will have to rebuild every DOM node in the repeat, even if two of the articles are the same.
```

Comment:
Does this mean that if there's any change to the array, Angular will re-render each DOM?